build.bat

```diff
@@ -0,0 +1,29 @@
+WHERE choco
+IF %ERRORLEVEL% NEQ 0 %PS% -NoProfile -ExecutionPolicy unrestricted -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET PATH=%PATH%;%systemdrive%\ProgramData\chocolatey\bin
+choco install sbt --acceptlicense -y
+WHERE sbt
+IF %ERRORLEVEL% NEQ 0 SET PATH=%PATH%;%systemdrive%\Program Files (x86)\sbt\bin
+choco install "\\columbus\sw_share\DEPARTMENT\SD Software\Java\jdk8.8.0.66.nupkg" --acceptlicense -y
```

Comment:
@haimizrael this is specific to your network?

The reason there is even a `build.sh` is so that people know `sbt clean editsource:edit assembly` should be run. We can add a `build.bat`, but I suppose anything more than calling sbt, and maybe the final `copy`, should not be in it. We can document that sbt and jdk are needed for the build.